server/reload.go

```diff
@@ -2589,3 +2627,34 @@ addLoop:
 
 	return add, remove
 }
+
+func diffProxiesTrustedKeys(old, new []*ProxyConfig) ([]string, []string) {
+	var add []string
+	var del []string
+	// Both "old" and "new" lists should be small...
+	for _, op := range old {
+		var found bool
```

Comment:
Mostly a nit but this could be `slices.Contains()`, same with below.

Reply:
But we don't want to compare the whole ProxyConfig, just the Key. Would slices.Contains() do only the compare on the op.Key?

Reply:
If it ends up gnarly then don't worry about it, but if it does end up being more concise, all the better!